src/components/Footer/index.js

```diff
@@ -68,24 +68,28 @@ export default function Footer () {
                 <a
                   aria-label="Link to Facebook"
                   href="https://www.facebook.com/TechLadies"
+                  target="_blank"
```

Comment:
```suggestion
                  target="_blank"
                  rel="noopener"
```
Can we also add this, it's better.